website/src/pages/Home/styles.ts

```diff
@@ -293,14 +293,23 @@ const useStyles = makeStyles((theme) => ({
             minWidth: '400px',
         },
     },
+    overrides: {
+        '.MuiButton-contained': {
+            background: theme.palette.background.paper,
+            width: '184px',
+        },
+        '.MuiButton-root': {
+            borderRadius: '8px',
+        }
+    },
     installationButton: {
```

Comment:
```suggestion
    installationButton: { 
         '&.MuiButton-contained' : { 
```
this should work, PTAL

Reply:
Thanks @rakeshPRaghu it worked!!